docs/setup/actions/parameter.md

```diff
@@ -0,0 +1,34 @@
+## Action parameters
+
+The `action.parameter` tag defines action parameters.
+Unless modified, default values are used for all parameters.
+Defaults are listed in [parameter.{{ include.implementation }}.xml](https://github.com/bagherilab/ARCADE/blob/main/src/arcade/{{ include.implementation }}/parameter.{{ include.implementation }}.xml).
+
+- Either or both `value` and `scale` attributes can be applied, with `value` applied first
+- Only numeric parameters can be modified using `scale`
+
+| ATTRIBUTE | DESCRIPTION                               |
+| --------- | ----------------------------------------- |
+| `id`      | parameter name                            |
+| `value`   | new parameter value                       |
+| `scale`   | scaling factor applied to parameter value |
+
+### Example: Modifying action parameters
+
+_The time delay parameter is set to the new value 2000._
+
+```xml
+<action.parameter id="TIME_DELAY" value="2000" />
+```
+
+_The default value of the time delay parameter is scaled by 2._
+
+```xml
+<action.parameter id="TIME_DELAY" scale="2" />
```

Comment:
I think this example doesn't quite work/is misleading because the default value for `TIME_DELAY` is 0, maybe we use `INSERT_RADIUS` or `INSERT_NUMBER`?